gef-ui/server/controllers/activities-controller/index.js

```diff
@@ -4,25 +4,35 @@ const { getApplication, getUserDetails } = require('../../services/api');
 
 // maps portalActivities array to create array in correct format for mojTimeline
 const mapPortalActivities = (portalActivities) =>
-  portalActivities.map((portalActivity) => ({
-    label: {
-      text: portalActivity.label,
-    },
-    text: portalActivity.text,
-    datetime: {
-      timestamp: fromUnixTime(new Date(portalActivity.timestamp)),
-      type: 'datetime',
-    },
-    byline: {
-      text: `${portalActivity.author.firstName} ${portalActivity.author.lastName}`,
-    },
-    html: portalActivity.html,
-    facilityType: portalActivity.facilityType,
-    ukefFacilityId: portalActivity.ukefFacilityId,
-    facilityId: portalActivity.facilityId,
-    maker: portalActivity.maker,
-    checker: portalActivity.checker,
-  }));
+  portalActivities.map(({ label, text, timestamp, author, html, facilityType, ukefFacilityId, facilityId, maker, checker }) => {
+    let bylineText = author.firstName;
+
+    if (author.lastName) {
+      bylineText += ` ${author.lastName}`;
+    }
```

Comment:
Whats your opinion on something like this - avoids having to use `let` and reassign the variable

```suggestion
    const bylineText = (`${author.firstName} ${author.lastName ?? ''}`).trim();
```

I think its still imperfect but I maybe prefer it?

Reply:
I think the let and condition is easier to read, but maybe my oldschool thinking 😄 

See what others think

Reply:
Sure happy to leave/change - I always find `let`s to be a bit of a codesmell but appreciate the reasoning here

Reply:
it is not the easiest to read.  I prefer the clunkier if statement